helm-charts/infisical/templates/mongodb-deployment.yaml

```diff
@@ -1,3 +1,4 @@
+{{- $mongodb := .Values.mongodb }}
 apiVersion: apps/v1
```

Comment:
Can we make this deployment optional (in case we want to use an external database)?

Reply:
I think what we can do is if `externalMongoDBConnectionString` is defined then no need to spin this deployment up. What do you think @Grraahaam